tests/dotnet-structuring.tests/LibraryTest.cs

```diff
@@ -60,11 +61,28 @@ public static IEnumerable<object[]> TemplatesGettingTested()
         }
 
         [Theory]
+        [Trait("Category", "UnitTest")]
         [MemberData(nameof(TemplatesGettingTested))]
-        public async Task TestTemplates(Template Template)
+        public async Task TestTemplatesAsync(Template Template)
+        {
+            CustomProcess process = new CustomProcess();
+            await TestTemplateAsync(Template.ShortName, process);
+            Assert.Equal("dotnet", process.StartInfo.FileName);
+            Assert.Contains($"new {Template.ShortName}", process.StartInfo.Arguments);
```

Comment:
kannst du sicher stellen, dass da auch immer nur die richtigen Attribute drin stehen ? 
wenn nein dann bitte genauer schauen was alles in den Arguments drin steht